### ISSUE TITLE
Updated Video Series

### ISSUE THREAD

**Issue by Tina-Chris** [inkstitch/inkstitch#1409]
Hi contributors
Some already heard and for others it will be new. I'm planning on making a new tutorial series for hands on approach to inkstitch, gradually introducing more difficulty.
I would like to have some input or a partner to pitch ideas back and forth :). So I'm thinking on the following, but would apprechiate a little confirmation or input. What is missing? Which order would be suitable? Can we make written instructions out of the tutorials to put on the homepage for persons rather reading than watching videos? @Augusa can I use the ideas from your blog for this video series? I'm refering especially to this overview https://lyogau.over-blog.com/2021/08/formation-a-inkstitch.html
@claudinepeyrat06 You made some tutos on the frenc FB group, Might I use those as well? Thinking on the snowflake, flowers and celtic knot? 

- Keep the installation guide and customization setup guide
    - I need help for MacOS; I have a Linux and a Win10 PC available
    - provide a link to inkstitch specific shortcuts and a list what they do
    - show nodes and node directions

- continous contour motif for running stitch includes explaination of:
    - dashed line symbolization
    - how to use the bezier tool
    - how to set up stitch length, bean stitch and back an forth stitch
    - show stitch out

- wide and small section satin motif includes explaination of:
    - conversion of outlines to satin lines
    - conversion of simple thick lines to satin lines (or later tutorial?)
    - how to set up stitch density and underlay
    - explain difference between thick single line and satin  (or later tutorial?)
    - show stitch out

- separated fill motiv 
    - auto digitize matrix image
    - use of break apart 
    - how to set up stitch density, directions and underlay
    - ordering or objects
    - show sitch out

- fill motiv with objects next to each other, outlined with bean stitch
    - explain push and pull
    - show stitch out with respecting push and pull and without
    - using start and stop signals

- combined motiv fill, satin and straight
    - explain push and pull

- making a custom text from the lettering module
    - choosing a font
    - font size guidelines
    - centering text
 
- digitizing satin text from the inkscape text 
    - splitting and combining nodes
    - push and pull, underlay

- digitizing a new font for inkstitch as explained here https://inkstitch.org/tutorials/font-creation/ 

- knock down stitch for towels 
    - explain use of Inkscape inset and outset function 
    - explain why its nicer to knock down
    - show stitch out of knockdown vs only watersoluble
    - travel along the edge setting

- Applique or doodle motif for children T-shirt
    - explain use of Inkscape inset and outset function 
    - explain why its necessary and show real live stitch out of knockdown vs only watersoluble

Idea dump:
- Fringe 
- Puff 
- keyfob
- patterns in satin
- patterns in fillstitch
- satin edges
- motiv along a path https://lyogau.over-blog.com/2019/10/inkstitch-essai-de-point-programmable.html 
- quilting stitch
- Interpolate
- custom color palette

**Comment by Tina-Chris** [comment]
@kaalleen Would you please tell me which pre-release version you find suitable to make the videos with?
Thanks, Tina

**Comment by Tina-Chris** [comment]
All right.
The menu will basically stay the same then?
I probably will not be that fast with the tutorials, anyway. I can make the
fill stitch ones later
Cheers, Tina


Kaalleen ***@***.***> schrieb am Di., 2. Nov. 2021, 18:50:

**Comment by Tina-Chris** [comment]
@kaalleen I tried to follow the instructions here https://inkstitch.org/docs/customize/#download-and-import-custom-shortcut-keys
But the xml file does not show in the List of "Tastaturkürzel Datei", instead the default is updated with the inkstitch specific shortcuts. 
Now that I write it I come to think that is the intention of the Mini-tutorial. Instead I would write the following, to make it more clear:
_Download and import custom shortcut keysPermalink

- Download the Ink/Stitch shortcut key file
- Go to Edit > Preferences > Interface > Keyboard
- Click on Import...
- Select your shortcut key file (inkstitch.xml)
- Click open

Now you will be able to use the shortcut keys described above._

I would write: **This imports the inkstitch specific shortcuts to the inkscape default.xml shortcut file, thereby adding the desired functionality.**

**Comment by kaalleen** [comment]
Ok :)

**Comment by Tina-Chris** [comment]
I just tried to use the shortcut file on linux. It did not overwrite the strg+shirft+L from the layers command to the  simulate. Should it? Thanks. Tina

**Comment by kaalleen** [comment]
Yes, it should. I tested it and although it was showing the correct keyboard shortcut in the list for the simulator it didn't overwrite the old command with the imported one. When I enter it by hand it asks if I want to replace. We have no influence on the behaviour though. This is an Inkscape issue.
